frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -14,104 +17,100 @@ message-base(v-bind='$props' @wrapperAction='action')
+      [PROPOSAL_VARIANTS.EXPIRED]: L('No members were added.')
     },
-    'remove-member': {
-      open: L('{from} wants to remove members from the group.', interaction),
-      passed: L('Left {title}', summary),
-      default: L('No members were removed.')
+    [PROPOSAL_REMOVE_MEMBER]: {
+      [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to remove members from the group.', options),
+      [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to remove members from the group.', options),
+      [PROPOSAL_VARIANTS.ACCEPTED]: L('Left {title}', options),
+      [PROPOSAL_VARIANTS.REJECTED]: L('No members were removed.'),
+      [PROPOSAL_VARIANTS.EXPIRED]: L('No members were removed.')
     },
-    'group-setting-change': {
+    [PROPOSAL_GROUP_SETTING_CHANGE]: {
       mincomeAmount: {
-        open: L('{from} wants to change the change the groups mincome. ', interaction),
-        passed: L('The groups mincome changed.'),
-        default: L('The group mincome hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the change the groups mincome. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the change the groups mincome. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups mincome changed.'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The group mincome hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The group mincome hasn\'t changed.')
       }
     },
-    'proposal-setting-change': {
+    [PROPOSAL_PROPOSAL_SETTING_CHANGE]: {
       votingRule: {
-        open: L('{from} wants to change the groups voting rules. ', interaction),
-        passed: L('The groups voting rules changed'),
-        default: L('The groups voting rules hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups voting rules. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the groups voting rules. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups voting rules changed'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The groups voting rules hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The groups voting rules hasn\'t changed.')
       },
       votingSystem: {
-        open: L('{from} wants to change the groups voting system. ', interaction),
-        passed: L('The groups voting system changed.'),
-        default: L('The groups voting system hasn\'t changed.')
+        [PROPOSAL_VARIANTS.CREATED]: L('{from} wants to change the groups voting system. ', options),
+        [PROPOSAL_VARIANTS.EXPIRING]: L('{from} wants to change the groups voting system. ', options),
+        [PROPOSAL_VARIANTS.ACCEPTED]: L('The groups voting system changed.'),
+        [PROPOSAL_VARIANTS.REJECTED]: L('The groups voting system hasn\'t changed.'),
+        [PROPOSAL_VARIANTS.EXPIRED]: L('The groups voting system hasn\'t changed.')
       }
     }
+  }[proposal.proposalType][proposal.variant]
```

Comment:
When I attempted to test the expiry locally by creating a proposal and modifying my computer's clock +13 days, I got this error here:
 
![Screen Shot 2022-10-19 at 6 20 24 PM](https://user-images.githubusercontent.com/138706/194446328-64cf1fec-97f8-4022-8224-24b1534322ba.jpg)

Reply:
@taoeffect, which proposal did you create on this?

Reply:
@Silver-IT I created a generic proposal.